app-crypt/zulucrypt/zulucrypt-5.4.0_pre20180223.ebuild

```diff
@@ -0,0 +1,51 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit cmake-utils vcs-snapshot
```

Comment:
Please don't use `vcs-snapshot` if not necessary. You have full commit id here, so `S` should be predictable.

Reply:
that was my recommendation

Reply:
From the eclass:

```
# Note: this eclass is no longer needed with the new-style 'archive'
# GitHub URLs. They have sane directory names and stable contents,
# so you should really prefer them.